migrator/k8s.go

```diff
@@ -0,0 +1,44 @@
+package migrator
+
+import (
+	"sort"
+
+	"github.com/giantswarm/microerror"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	labelMasterID = "giantswarm.io/master-id"
+)
+
+func createK8SClient() (kubernetes.Interface, error) {
+	// creates the in-cluster config
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+	// creates the clientset
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	return clientset, nil
+}
+
+// getNodeNames return nodeName list ordered by master id label.
+func getNodeNames(nodes []v1.Node) []string {
```

Comment:
Same here, this thing deserves unit tests IMO.

Reply:
done